src/app/shared/mock-data.service.ts

```diff
@@ -186,6 +192,15 @@ export class MockDataService {
     return this.makeCopy(this.workItemLinks);
   }
 
+  public getWorkItemLinksForWorkItem(workItemId: string): any {
+    var result: any = [];
+    for (var i = 0; i < this.workItemLinks.length; i++)
+      if (this.workItemLinks[i].relationships.source.data.id === workItemId) {
```

Comment:
Mmmh..I discussed that with Naina. We came to the conclusion that only the source should be relevant. Are you sure? Because that would mean that when I am looking for the links for a work item, I would also get all links that are pointing "towards" that item, not only outgoing links..